layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationAddNodeDrawer.vue

```diff
@@ -161,6 +167,12 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 export default {
   data () {
     return {
+      startPublicationDate: null,
+      endPublicationDate: null,
+      startPublicationDateTime: null,
+      endPublicationDateTime: null,
+      startPostDate: null,
+      endPostDate: null,
```

Comment:
```suggestion
      startScheduleDate: null,
      endScheduleDate: null,
      startScheduleDateTime: null,
      endScheduleDateTime: null,
```
I think that we can minimise the params